### PR TITLE
[Story/CLAUDIA-5615] Add/Update elapsed time as context attribute of the region

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers

@jesuspg @pratid 
#### Description

The elapsed time information is calculated by the own script executed on Jenkins because the 'duration' of the Jenkins' job only can be consulted when it has finished. To avoid retrieving this data from the Dasboard and reduce the necessary logic, the measure of this time will be performed by the script. This way this attribute will be stored as a context attribute to be consulted by anyone.
#### Testing

Locally and Jenkins.

### CODE REVIEW

**Comment by flopezag** [comment]
LGTM

File: fiware-region-sanity-tests/doc/publish_status_and_test_data.rst [hunk 1]
```diff
@@ -71,6 +71,16 @@ Here is an example of the generated ``test_results.txt`` for an specific region
...
 
 
+Elapsed execution time of Sanity Checks
+---------------------------------------
+
+Apart from the context attributes described above, when Sanity Cheks
+of a node have been executed from Jenkins, a new attribute is updated in the
+context information of that region: the elapsed time of the execution
+``sanity_checks_elapsed_time``. This attribute will have the time in
+milliseconds or 'N/A' if Sanity tests have not finished yet.
```
Comment (jesuspg) [inline fiware-region-sanity-tests/doc/publish_status_and_test_data.rst:81]: sanity_check(remove S)_elapsed_time

Comment (jframos) [inline fiware-region-sanity-tests/doc/publish_status_and_test_data.rst:81]: fiexed in 7a737a5

**Comment by pratid** [comment]
LGTM

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**